src/avram/migrator/drop_index_statement.cr

```diff
@@ -53,4 +53,14 @@ class Avram::Migrator::DropIndexStatement
       return [columns]
     end
   end
+
+  private def index_name
+    if @name
+      @name
+    elsif @columns
+      "#{@table}_#{columns.join("_")}_index"
+    else
+      raise "No name or columns specified for drop_index"
```

Comment:
I believe you can remove this if the the initializers ensure that one or the other is set since this will never be hit